### DISCUSSION TITLE
2023-10-17

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#670] [daily_meeting]
### 개인 일정
히이로 - 3~4시 면담

### 어제의 회고
폴링 관련 QA

### 오늘의 할 일
- 프론트, 백 오전 강의 
- 백엔드 무중단 배포 검토 및 필요 스크립트 설계
- 최종 데모데이 요구사항 검토

### 내일의 할 일
수요일까지 데모데이 소개 페이지 팀별 페이지

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 아침에 일어날 때마다 머리가 너무 아프네요... 그래도 할 수 있는 것을 해야죠

### 체크아웃
8점 - 누가 미션이랑 무중단 배포 좀 해주면 좋겠네요

**Comment by nlom0218** [comment]
### 체크인

5점 - 백준 문제 왤케 어렵죠?

### 체크아웃

10점 - 캐시 재밌는데, CacheStorage를 사용하지 못한 것이 아쉽네요ㅠㅠ

**Comment by jaehee329** [comment]
### 체크인
6점 - 항상 할거많네요
### 체크아웃
5점 - 저녁에 집에 가고싶어요

**Comment by aak2075** [comment]
### 체크인

10점 - 왜 안풀리죠

### 체크아웃

10점 - 시간은 참 빨라

**Comment by woosung1223** [comment]
### 체크인
8점 - 오전에 강의가 있네요 귀-찮

### 체크아웃
7점 - 확실히 데모데이가 가까워지니까 할게 많아지네요 🥲

**Comment by yeopto** [comment]
### 체크인

10점 - 오늘도 활기차게 시작해봅니다.

### 체크아웃

10점 - 오늘도 하루가 빨리가서 좋네요 그만큼 알차게 보냈다는거겠죠?

**Comment by woo-jk** [comment]
### 체크인
9점 - 안피곤해요

### 체크아웃
10점 - 하루가 빨라요
